src/workloads/scale/NegativeScalingLoadStress.yml

```diff
@@ -1,122 +0,0 @@
-SchemaVersion: 2018-07-01
```

Comment:
I think this is the Tenemos workload ([PERF-4749](https://jira.mongodb.org/browse/PERF-4749)) that is likely to be included in the scope of [SPM-3676](https://jira.mongodb.org/browse/SPM-3676) - are we sure we want to delete this?

Reply:
There's a discussion about it [here in the doc](https://docs.google.com/document/d/1D8YfLNVpZ_KT6O0OrXouCbq6vOm-0YJgwE3KqrAKg6U/edit?disco=AAABVlKKlE4).

I feel we will and should pick another workload as our standard for detecting negative scaling issues.

Reply:
We're currently scoping that project, and Francisco mentioned using this Tenemos workload in [PERF-5760](https://jira.mongodb.org/browse/PERF-5760). Let me ask on the scope about this because I don't think a decision was fully met in that discussion.